manga_translator/manga_translator.py

```diff
@@ -1,14 +1,23 @@
 import asyncio
+import base64
+import io
+
+import PIL
 from PIL import Image
 import cv2
+import langid
 import numpy as np
 import requests
 import os
 import torch
 import time
 import logging
+import nest_asyncio
+from aiohttp import web
+from marshmallow import Schema, fields, ValidationError
 
 from .args import DEFAULT_ARGS
+from .server.web_main import MAX_IMAGE_SIZE_PX, VALID_TRANSLATORS, VALID_DETECTORS, VALID_LANGUAGES, VALID_DIRECTIONS
```

Comment:
I think in this case you shouldnt import them but define them inside the class.

Reply:
I am using now the same imports as in args.py. It wouldnt make sense to redifine them, because they are just overwriting the default arguments.